eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -202,7 +202,7 @@ jobs:
         inputs:
           searchFolder: "$(System.DefaultWorkingDirectory)/sdk"
           testResultsFiles: "$(TestResultsFiles)"
-          testRunTitle: "$(OSName) - $(TestType) - Integration Tests - [Node $(NodeTestVersion)]"
+          testRunTitle: "${{ parameters.ServiceDirectory }} ${{ parameters.CloudConfig.Cloud }} $(Agent.JobName) $(TestType)"
```

Comment:
Is it still interesting to include the test version?

Reply:
Ah, I misunderstood `TestType` as `client|data` but that's actually `SDKType`. The former is already included in `Agent.JobName` since it's in the matrix, so I'll remove it from the end of the title.